virttest/utils_secret.py

```diff
@@ -0,0 +1,59 @@
+"""
+Secret related utility functions
+"""
+
+import logging
+
+from virttest import virsh
+
+
+def get_secret_list(remote_virsh=None):
+    """
+    Get secret list by virsh secret-list from local or remote host.
+
+    :param remote_virsh: remote virsh shell session.
+    :return secret list.
+    """
+    logging.info("Get secret list ...")
+    try:
+        if remote_virsh:
+            secret_list_result = remote_virsh.secret_list()
+        else:
+            secret_list_result = virsh.secret_list()
+    except Exception as e:
+        logging.warn("Exception thrown while getting secret lists: %s", str(e))
+
+    secret_list = secret_list_result.stdout_text.strip().splitlines()
```

Comment:
If you can not get secret_list_result, then the following operation also will raise error, I think it is better to move them also in try.

Reply:
if error is thrown when secret_list = secret_list_result.stdout_text.strip().splitlines() , it is anticipated and the logic need break here.